pages/berita/[slug].tsx

```
@@ -91,10 +94,41 @@ export default function ReadNews({
+                <span
+                  key={cat.slug}
+                  className='inline-block bg-[#4FBEBC] mr-2 mb-2 px-3 py-1 rounded-full font-semibold text-white text-xs'>
+                  {cat.title}
+                </span>
+              ))}
+            </div>
+            <div className='col-span-12 px-5 xl:px-0'>
+              <div className='gap-3 grid grid-cols-4'>
+                {latestNews &&
+                  latestNews.map((news) => (
+                    <div className='gap-3 border-slate-200 col-span-1 bg-white p-3 border'>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)